helpers.js

```diff
@@ -98,6 +100,17 @@ var Helpers = {
 		return version.replace('-pre', '');
 	},
 
+	getHomePage: function () {
+		var originUrl = execSync('git config --get remote.origin.url');
+		originUrl = String(originUrl);
```

Comment:
You can change this to:
```js
var originUrl = this.exec('git config --get remote.origin.url');
```

Note that `this.exec()` will return a `String` value.